cs3api4lab/utils/file_utils.py

```diff
@@ -37,3 +37,14 @@ def calculate_content_size(content):
             content_len = len(content.decode('utf-8'))
         content_size = str(content_len)
         return content_size
+
+    @staticmethod
+    def get_user_directory(config):
```

Comment:
@diocas  If you can generate a path to the user directory and pass it into the plugin config we can skip this part with generating the path by ourselves. user directory should still be optional or contain the same value as home_dir in case of other environments especially our local dev environments

Reply:
Yes, I can generate the path of the user and put it in the config. So please skip this part and return it as is.
My only issue is that this is not properly configurable, it's a hardcoded path in the end.